README.md

```diff
@@ -0,0 +1,75 @@
+<p align="center">
+ <img src="https://user-images.githubusercontent.com/62607343/202245103-044d5c0b-7dec-416b-a178-66b4875dc399.png" width="200" alt="SAIL Logo" /></a>
+</p>
+
+
+
+# NOTIFICATION-SERVICE
+
+This is a notification microservice that lets you create email templates, queue your email on AWS SQS, and send the email to users of your applications through AWS SES.
+
+
+## Packages
+<li> <a href="https://github.com/hicsail/notification-service/tree/readMe/packages/server/src/email"> Email Notification Service </a>
+<li> <a href="https://github.com/hicsail/notification-service/tree/readMe/packages/server/src/templates"> Email Template Service </a>
+<li> <a href="https://github.com/hicsail/notification-service/tree/readMe/packages/server/src/health">  Health Service </a>
+
+## 📖 Prerequisite
+
+In order to run/develop this Microservice, your project needs the following dependecies:
+
+<pre>
+"node": "^14.15.0 || >=16.10.0",
```

Comment:
NIT: Its probably best to refer the user back to the `package.json` since this section is less likely to be updated to match the exact requirements as things change